src/docs/admin/references/ServiceNowReference.adoc

```diff
@@ -0,0 +1,560 @@
+*Product Name:* ServiceNow +
+*Product Version:* Quebec +
+*Compatibility:* Tokyo
+
+==== Overview
+
+The ServiceNow integration provides means to establish a bidirectional communication
+between a ProActive server, and a ServiceNow instance.
+
+==== Objectives
+
+The goal is to provide templates and coding elements to help ServiceNow and ProActive engineers to easily integrate both solutions in their pipelines.
+
+It boils down to: +
+
+* Provide reusable code in ServiceNow to query ProActive's REST APIs.
+* Provide workflows in ProActive's catalog to query ServiceNow's REST APIs.
+
+==== Installation
+
+This section describes the available processes to set up the required files in ServiceNow in order to communicate
+with ProActive. The process choice is dictated by the architecture the integration will have in ServiceNow. Because ServiceNow
+is application-based, the natural course of action is to create a dedicated ProActive application.
+Otherwise, you might want to incorporate this integration in another application that already exists. In such case,
+you will have to manually create files in the host application, which means copy-paste code that will be presented in the next sections.
+
+===== Import from source control
+
+Working from an online repository is the most convenient and durable way to proceed.
+Importing an application from source control will result in a new application being created in this ServiceNow instance based on the remote repository you specify.
```

Comment:
will result in a new application to be created in this ServiceNow instance which is based on the specified remote repository